javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/bootstrap/internal/CommonConfig.java

```diff
@@ -41,6 +42,8 @@ public static CommonConfig get() {
         config.getList("otel.instrumentation.http.capture-headers.server.request", emptyList());
     serverResponseHeaders =
         config.getList("otel.instrumentation.http.capture-headers.server.response", emptyList());
+    rpcRequestMetadata =
+        config.getList("otel.instrumentation.rpc.capture-metadata.request", emptyList());
```

Comment:
Is there any need to distinguish between client and server request metadata? Or is it fine to just capture both?

Reply:
@samimusallam 
What is your opinion?

Reply:
@mateuszrzeszutek 
The spec currently mentions the request and the response, and no reference to the client or the server.

Reply:
@samimusallam What is your opinion on the matter?

Reply:
Request and response metadata should be captured for both server and client